configure.py

```diff
@@ -453,6 +453,7 @@ def find_headers(repodir, excluded_dirs):
     'test/boost/restrictions_test',
     'test/boost/role_manager_test',
     'test/boost/row_cache_test',
+    'test/boost/rust_test',
```

Comment:
I don't see this source file in this commit.

Reply:
Oof, I missed it when adding to the commit, I'm adding it in the rebase